components/Map/index.js

```diff
@@ -395,13 +400,13 @@ class MapContainer extends React.Component {
           initialState={this.initialState}
         />
         <CookieNotice />
-        {(isDbLoading || isMapLoading) ? (
+        {(isDbLoading || isMapLoading) && acknowledgeModalShown === false ? (
```

Comment:
This actually disable the `LoadingModal` and `FunFactModal` after the initiial load, which is not what we want. Our goal is to disable `AcknowledgeModal` if it already has been rendered once on initial load.

We should leave line 403 as it was: 
```
{(isDbLoading || isMapLoading) ? (
```